x/qgb/orchestrator/test/test_utils.go

```diff
@@ -90,6 +90,6 @@ func verifyOrchestratorDcSignature(broadcasted sdk.Msg, dc types.DataCommitment)
 	return nil
 }
 
-func commitmentFromQuery(query string) []byte {
-	return tmhash.Sum([]byte(query))
+func commitmentFromQuery(beginBlock uint64, endBlock uint64) []byte {
```

Comment:
[question] is this function name still appropriate given the parameter change?

Reply:
thanks, changed.